pytest.ini

```diff
@@ -7,7 +7,7 @@
 asyncio_mode = auto
 
 # jupyter_server plugin is incompatible with notebook imports
-addopts = -p no:jupyter_server -m 'not selenium' --color yes --durations 10 --verbose
+addopts = -p no:jupyter_server -m 'not selenium and not playwright' --color yes --durations 10 --verbose
```

Comment:
also here: _rename_ selenium to more generic `browser` rather than adding a new playwright mark

Reply:
thank you, the changes have been done